resource-managers/yarn/src/main/scala/org/apache/spark/scheduler/cluster/YarnClientSchedulerBackend.scala

```diff
@@ -121,7 +123,8 @@ private[spark] class YarnClientSchedulerBackend(
         allowInterrupt = false
         sc.stop()
       } catch {
-        case e: InterruptedException => logInfo("Interrupting monitor thread")
+        case _: InterruptedException | _: InterruptedIOException
+        => logInfo("Interrupting monitor thread")
```

Comment:
Move `=>` to previous line.

Reply:
that will cause scalastyle error for too long line

Reply:
I believe @mridulm meant:
```
        case _: InterruptedException | _: InterruptedIOException =>
          logInfo("Interrupting monitor thread")
```